examples/ephemeral-spark/hbase-cluster.tf

```
@@ -47,15 +47,15 @@ module "emr-hbase" {
   core_instance_on_demand_count   = 4
   # core_instance_spot_count    = 4
   # core_bid_price_as_percentage_of_on_demand_price = 100
-  master_instance_type = "m4.large"
-  core_instance_type   = "r5.xlarge"
+  master_instance_type = "m6g.xlarge"
+  core_instance_type   = "m6g.xlarge"
```

The r6g.xlarge has double the memory of the m6g.xlarge for marginally higher cost.

```suggestion
  core_instance_type   = "r6g.xlarge"
```